tests/unit/test_sap_rfc.py

```diff
@@ -0,0 +1,190 @@
+from collections import OrderedDict
+
+from viadot.sources import SAPRFC
+
+import pytest
+import pandas as pd
+
+
+@pytest.fixture(scope="session")
+def sap():
+    sap = SAPRFC(config_key="sap_rfc")
+    yield sap
+
+
+sql1 = "SELECT a AS a_renamed, b FROM table1 WHERE table1.c = 1"
+sql2 = "SELECT a FROM fake_schema.fake_table WHERE a=1 AND b=2 OR c LIKE 'a%' AND d IN (1, 2) LIMIT 5 OFFSET 3"
+sql3 = "SELECT b FROM c WHERE testORword=1 AND testANDword=2 AND testLIMITword=3 AND testOFFSETword=4"
+sql4 = "SELECT c FROM d WHERE testLIMIT = 1 AND testOFFSET = 2 AND LIMITtest=3 AND OFFSETtest=4"
+sql5 = sql3 + " AND longword123=5"
+sql6 = "SELECT a FROM fake_schema.fake_table WHERE a=1 AND b=2 OR c LIKE 'a%' AND d IN (1, 2) AND longcolname=3 AND otherlongcolname=5 LIMIT 5 OFFSET 3"
+sql7 = """
+SELECT a, b
+FROM b
+WHERE c = 1
+AND d = 2
+AND longcolname = 12345
+AND otherlongcolname = 6789
+AND thirdlongcolname = 01234
+LIMIT 5
+OFFSET 10
+"""
+sql8 = "INSERT INTO customers name VALUES 'John Doe'"
+sql9 = " SELECT * FROM table1"
+sql10 = "SELECT a FROM fake_schema.fake_table WHERE a=1 AND b=2 OR c LIKE 'a%' AND d IN (1, 2) AND longcolname=3 OR otherlongcolname=5 LIMIT 5 OFFSET 3"
+sql11 = "SELECT col FROM table1, table2"
+
+
+def test_get_function_parameters(sap):
+    assert sap.get_function_parameters("RFC_READ_TABLE", description=None) == [
+        "ET_DATA",
```

Comment:
Same values as above in the assert. Why not bring together and make a more human readable?